string text
---
bool affirmation
bool negation
string clientName
string[] clientOrder
